File: Array/zhanchengguo/0623-统计位数为偶数/Main.java
package com.test.ktapplication;


import android.util.Log;

public class Main {

    void main() {
        int nums[] = {12, 345, 2, 6, 7896};
        int result = findNumbers(nums);
        Log.e("result is", result + "");
    }

    public int findNumbers(int[] nums) {
        int count = 0;
        for (int num : nums) {
            if (String.valueOf(num).length() % 2 == 0) {
                count++;
            }
        }
        return count;
    }
}

File: Array/zhanchengguo/0623-统计位数为偶数/main.kt
package com.test.ktapplication

fun main() {
    val s = arrayOf(12, 345, 2, 6, 7896)
    val result = findNumbers(s)
    print("result is ${result}")
}

fun findNumbers(nums:Array<Int>):Int{
    var count = 0
    for (num in nums) {
        if (num.toString().length % 2 == 0) {
            count++
        }
    }
    return count
}